liblxqt-config-cursor/selectwnd.cpp

```diff
@@ -37,22 +38,24 @@
 
 #define HOME_ICON_DIR QDir::homePath() + "/.icons"
 
-SelectWnd::SelectWnd(LXQt::Settings* settings, QWidget *parent) : QWidget(parent), mSettings(settings)
+SelectWnd::SelectWnd(LXQt::Settings* settings, QWidget *parent)
+    : QWidget(parent),
+      mSettings(settings),
+      ui(new Ui::SelectWnd)
```

Comment:
Hm...actually, the `Ui::SelectWnd` object gets never destroyed. We should use a smart pointer here.